README.md

```diff
@@ -1,2 +1,65 @@
+[![GitHub Repos](https://img.shields.io/badge/dynamic/json?style=flat-square&logo=github&logoColor=white&label=Repos&labelColor=24292e&color=8B5CF6&query=%24.public_repos&url=https%3A%2F%2Fapi.github.com%2Forgs%2Fpluto&scale=1.5)](https://github.com/pluto)
+</div>
+
+
+Pluto is a set of developer tools that makes applied cryptography accessible to developers across the world. 
+
+Our first tools help developers incorporate Web Proofs into their applications. Web Proofs are a new type of zero-knowledge proof that enable verifiable access to Web2 data sources.
+
+For more information on Pluto, see our [documentation](https://docs.pluto.xyz), [website](https://pluto.xyz), or follow us on [Twitter](https://twitter.com/plutolabs_).
+
+
+## Web Proofs 
+
+The Web Proofs SDK consists of three main components:
+1. Pluto CLI
+2. Pluto web SDK
+3. Pluto mobile SDK
+
+Developers can generate Web Proofs on demand using our SDK/CLI. At launch, this platform supports proofs for OAuth signatures, JSON Web tokens, Venmo transaction history, Reddit posting history, and Amazon purchase history.
+
+## Main repositories
+
+- [Ronkathon](https://github.com/pluto/ronkathon): Educational project for learning about applied cryptography; a Rust implementation of a collection of cryptographic primitives.
+- [Web Prover](https://github.com/pluto/web-prover): [Description of Web Prover repo]
+- [AES Proof](https://github.com/pluto/aes-proof): Implementation of AES-GCM in Circom.
+- [Extractor](https://github.com/pluto/extractor): Tool for extracting and processing Web2 data for Web Proofs
+- [Web Proofs](https://github.com/pluto/web-proofs): A TypeScript library designed to help applications generate Web Proofs for seamless integration with our iOS mobile app and Chrome extension
+- [Attest Integrations](https://github.com/pluto/attest-integrations): A container for Web Proof target domain integrations.
+  
+## Contributing
+
+We welcome contributions to our open-source projects. If you want to contribute or follow along with contributor discussions, join our [main Telegram channel](https://t.me/pluto_xyz/1) to chat about Pluto's development.
+
+Our contributor guidelines can be found in [CONTRIBUTING.md](./CONTRIBUTING.md). A good starting point is issues labelled 'bounty' in our repositories.
+
+For now, we're inviting collaborators to participate in [Ronkathon](https://github.com/pluto/ronkathon) educational project. In the coming weeks, we will be open-sourcing other core product repositories, which will also have open issues for contributors.
+
+## Getting help
+
+
+Your question might already be answered in our [docs](https://docs.pluto.xyz) or on our FAQ. If not:
+
+- Join our [Telegram](https://t.me/pluto_xyz/1) to get help and participate in discussions
+- Open a discussion [link to discussions]
+- Open an issue [link to issues]
+
+  
+## Acknowledgements
+
+TLSN team
+Origo
+Electron Labs, aes proof
+python plonkathon
+plonk-by-hand
```

Comment:
```suggestion
```
This is great but i don't think this is the right spot for this

Reply:
whats the right spot for it then? just want to make sure we dont get accused of not attributing appropriately

Reply:
I think that it is a per repository thing to site the code related and follow license requirements